internal/resource/config/administrativeaccount/administrative_account_data_source.go

```diff
@@ -19,7 +19,7 @@ var (
 )
 
 // Create a Administrative Account data source
-func NewAdministrativeAccountDataSource() datasource.DataSource {
+func AdministrativeAccountDataSource() datasource.DataSource {
```

Comment:
Why are we changing all these?

Reply:
to match the resource format - no need to have "New"